db/migrations/00028_create_header_sync_logs_table.sql

```diff
@@ -17,6 +17,13 @@ CREATE TABLE header_sync_logs
     UNIQUE (header_id, tx_index, log_index)
 );
 
+CREATE INDEX header_sync_logs_address
+    ON header_sync_logs (address);
```

Comment:
Not introduced by this PR (nor something that needs to be addressed by it), but I'm wondering if `address` should be renamed to `address_id`

Reply:
Yeah that makes sense. I'll just do it in this PR

Reply:
Actually nvm, I didn't consider that would entail changes to the transformer repo as well 😅 but I'll take care of it later